inference/huggingface/text-generation/token-generator.py

```
@@ -0,0 +1,59 @@
+]
+
+outputs = generator(inputs,
+        max_new_tokens=args.max_new_tokens
+        )
+
+if args.local_rank == 0:
+    for i, o in zip(inputs, outputs):
+        print(f"\nin={i}\nout={o}\n{'-'*60}")
+    average_num_tokens = 0
+    for o in outputs:
+        for key, value in o[0].items():
```

Ruff (0.8.2)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)